tasks/shell_pcs/pcs-cib-resource-primitive.yml

```diff
@@ -65,14 +65,14 @@
     cmd: >
       pcs -f {{ __ha_cluster_tempfile_cib_xml.path | quote }}
       -- resource utilization {{ resource.id | quote }}
-      {% if resource.utilization | d([]) %}
```

Comment:
Why do you remove this `if`?

Reply:
I moved it to `when` statement. If there is no utilization defined, we don't want the command to be run at all.